test/Libraries/RevitTestServices/RevitSystemTestBase.cs

```diff
@@ -229,9 +230,25 @@ protected override void StartDynamo(TestSessionConfiguration testConfig)
                 // SystemTestBase. That pathResolver will be used only in StartDynamo
                 // of the base class, here a local instance of pathResolver is used.
                 // 
-                var revitTestPathResolver = new RevitTestPathResolver();
+                var commandData = new DynamoRevitCommandData(RevitTestExecutive.CommandData);
+                var userDataFolder = Path.Combine(Environment.GetFolderPath(
+                  Environment.SpecialFolder.ApplicationData), 
+                  "Dynamo", "Dynamo Revit");
+                var commonDataFolder = Path.Combine(Environment.GetFolderPath(
+                  Environment.SpecialFolder.CommonApplicationData),
+                  "Autodesk", "RVT " + commandData.Application.Application.VersionNumber, "Dynamo");
+
+                // Set Path Resolver's user data folder and common data folder with DynamoRevit runtime.
+                var pathResolverParams = new TestPathResolverParams()
+                {
+                   UserDataRootFolder = userDataFolder,
+                   CommonDataRootFolder = commonDataFolder
+                };
+                RevitTestPathResolver revitTestPathResolver = new RevitTestPathResolver(pathResolverParams);
```

Comment:
Is it possible that direct get pathResolverParams inside the constructor of RevitTestPathResolver since that RevitTestPathResolver already has key word 'Revit'?